eurocropsml/acquisition/region.py

```diff
@@ -139,7 +143,8 @@ def add_nuts_regions(
     )
 
     for month in tqdm(
-        range(config.months[0], config.months[1] + 1), desc="Adding NUTS regions to data..."
+        range(1, 8)
+        # range(config.months[0], config.months[1] + 1), desc="Adding NUTS regions to data..."
```

Comment:
Was this changed to fixed values only for debugging?

Reply:
Yes, sorry! Removed.